bff-service-nest/.env

```diff
@@ -0,0 +1,3 @@
+PORT=3000
+product=https://320ihl0yyc.execute-api.eu-west-1.amazonaws.com/dev
+cart=http://max-korsakov-cart-api-development.eu-west-1.elasticbeanstalk.com/api/profile
```

Comment:
It is bad practice to publish a .env to github

Reply:
O, thx, seems forgot to add it to gitignore